__tests__/app.test.js

```diff
@@ -152,7 +152,77 @@ describe("/api", () => {
                         })
                     })
                 })
+                describe("POST", () => {
+                    test("POST /comments should return with a newly added comment", () => {
+                        return request(app)
+                        .post("/api/articles/3/comments")
+                        .send({
+                            author: "Jormanji",
```

Comment:
Requests should be sent with an object including 'username' key instead of 'author'.